udata_gouvfr/theme/templates/banners/banner-2020-06-datactivist-survey.html

```diff
@@ -0,0 +1,30 @@
+<nav class="navbar navbar-static-top navbar-subnav">
+
+    <div class="container banner">
+
+        <p class="text-center text-white">
+          Les portails 
+          <a 
```

Comment:
remove the link (useless since we're already there, and it creates distraction from the main CTA)